templates/portfolio/styles.css

```diff
@@ -0,0 +1,177 @@
+/*body:after {
```

Comment:
This looks like some test stuff left behind, so you should probably remove it. If it's here for a reason, though, please add a comment explaining why the user might want to enable it.